eng/setup-mongo.sh

```diff
@@ -0,0 +1,22 @@
+cd ../Meadowlark-js/backends/meadowlark-mongodb-backend/docker/
```

Comment:
This script is currently not being used, but I consider it can be useful for getting started instructions to setup the local environment from scratch. I can leave it in the PR or add it in another one.

Reply:
Can leave it